dipy/data/fetcher.py

```diff
@@ -631,8 +635,8 @@ def fetch_cenir_multib(with_raw=False):
                          '4e4324c676f5a97b3ded8bbb100bf6e5'])
 
     files = {}
-    baseurl = \
-'https://digital.lib.washington.edu/researchworks/bitstream/handle/1773/33311/'
+    baseurl = 'https://digital.lib.washington.edu/\
```

Comment:
no need to split url on two lines, even fi they go off at least they are easy to select and copy/paste.